fontir/src/ir.rs

```diff
@@ -664,6 +664,20 @@ impl GlobalMetrics {
         }
     }
 
+    pub fn set_if_some_or_fallback(
+        self: &mut GlobalMetrics,
```

Comment:
I'm... not sure why this is like this above, but this signature is just a weird way of writing `&mut self` as the first param, and we should stick to that.

(unless I'm missing something)

Reply:
I just copied the above function and added a new parameter without giving it much thought.